### PR TITLE
sync favicon

### LINKED ISSUE #7294: Bookmark Favicons are not sync
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
Bookmark Favicons are not sync

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 x64

- Brave Version (revision SHA):
Brave	0.13.5
rev	59a41eb

- Steps to reproduce:
    1. Enable Sync on two devices
    2. Import bookmarks on Pyramid 1
    3. Bookmarks synced on Pyramid 0 doesn't have favicons
    
- Actual result:
Favicons missing in pyramid 0 after bookmarks sync

- Expected result:
Favicons should be shown on both pyramid 0 and pyramid 1

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
N/A

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:
Pyramid 0
image
Pyramid 1
image

- Any related issues:
cc: @ayumi

### PR DESCRIPTION
**Pull request by diracdeltas**
Test Plan:
1. npm start
2. enable sync in pyramid 0
3. npm run start2
4. sync pyramid 1 to pyramid 0
5. bookmark google.com in pyramid 0
6. in pyramid 1, go to about:bookmarks. you should see the google favicon.

Fix #7294

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "preload-httpse": "node ./preload-httpse.js",
     "start-log": "node ./tools/start.js --user-data-dir=brave-development --debug=5858 --enable-logging=stderr --v=1 --enable-extension-activity-logging --enable-sandbox-logging --enable-dcheck",
     "start": "node ./tools/start.js --user-data-dir=brave-development --debug=5858 --enable-logging --v=0 --enable-extension-activity-logging --enable-sandbox-logging --enable-dcheck",
+    "start2": "node ./tools/start.js --user-data-dir=brave-development-2 --debug=5859 --enable-logging --v=0 --enable-extension-activity-logging --enable-sandbox-logging --enable-dcheck",
```
Comment (ayumi) [inline package.json]: start2

Comment (diracdeltas) [inline package.json]: ?

Comment (diracdeltas) [inline package.json]: yah i figured it is useful to testers now that sync is merged

**Comment by diracdeltas** [comment]
thx

**Pull request merged**